openhands/server/routes/manage_conversations.py

```diff
@@ -501,10 +502,21 @@ async def get_prompt(
         # placeholder for error handling
         raise ValueError('Settings not found')
 
+    env_base_url = os.environ.get('LLM_BASE_URL')
```

Comment:
```suggestion
```

We don't use global envs for settings (except for the resolver); `settings.llm_base_url` should be enough!